src/libs/actions/App.js

```diff
@@ -59,8 +79,14 @@ AppState.addEventListener('change', (nextAppState) => {
     appState = nextAppState;
 });
 
+function triggerUpdateAvailable() {
+    Onyx.merge(ONYXKEYS.UPDATE_AVAILABLE, true);
```

Comment:
Couldn't this be `Onyx.set(` since we're only storing a bool?

Reply:
Yeah, I think that's correct!

Reply:
https://github.com/Expensify/App/pull/7828